MogglesClient/PublicInterface/Notifications/NotificationService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using Newtonsoft.Json;
+
+namespace MogglesClient.PublicInterface.Notifications
+{
+    public class NotificationService : INotificationService
+    {
+        private readonly IMogglesConfigurationManager _mogglesConfigurationManager;
+
+        public NotificationService(IMogglesConfigurationManager mogglesConfigurationManager)
+        {
+            _mogglesConfigurationManager = mogglesConfigurationManager;
+        }
+
+        public void TryNotifyMissingFeatureToggle(string featureFlagName)
+        {
+            var webHook = _mogglesConfigurationManager.GetNotificationWebHook();
+
+            if (string.IsNullOrEmpty(webHook))
+                return;
+
+            using (var client = new HttpClient {BaseAddress = new Uri(webHook)})
```

Comment:
I would wrap this in a try catch in case the call fails for some reason. 
We don't want the LIVE apps to throw exceptions (this has happened in the past with the calls that retrieve the feature toggles)